src/phoenix/auth.py

```diff
@@ -86,40 +86,42 @@ class _PasswordRequirements:
     upper_case: bool = False
     lower_case: bool = False
 
-    def validate(self, password: str) -> None:
+    def validate(self, string: str, err_msg_subject: str = "Password") -> None:
```

Comment:
```suggestion
    def validate(self, string: str, err_msg_subject: str = "Password") -> None:
```
Switch to kwargs? signature string, string might be a bit vague